src/main/kotlin/org/jetbrains/research/testspark/collectors/data/Technique.kt

```diff
@@ -0,0 +1,8 @@
+package org.jetbrains.research.testspark.collectors.data
+
+/**
+ * Enum representing different techniques.
+ */
+enum class Technique {
+    LLM, EA
+}
```

Comment:
Please, rename the `EA` to what is stands for. What the abbreviation means needs to be clarified.

Reply:
@pderakhshanfar, what will you suggest. Maybe, `ES` -- EvoSuite? Or `SB` -- search-based?